implementations/micrometer-registry-prometheus/src/test/java/io/micrometer/prometheus/PrometheusMeterRegistryTest.java

```diff
@@ -646,23 +647,107 @@ void openMetricsScrapeWithExemplars() {
         Counter counter = Counter.builder("my.counter").register(registry);
         counter.increment();
 
-        assertThat(registry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100))
-                .startsWith("# TYPE my_counter counter\n")
-                .contains("# HELP my_counter  \n")
-                .contains("my_counter_total 1.0 # {span_id=\"testSpanId\",trace_id=\"testTraceId\"} 1.0")
-                .endsWith("# EOF\n");
+        Timer timer = Timer.builder("test.timer")
+                .serviceLevelObjectives(Duration.ofMillis(100), Duration.ofMillis(200), Duration.ofMillis(300))
+                .register(registry);
+        timer.record(Duration.ofMillis(15));
+        timer.record(Duration.ofMillis(150));
+        timer.record(Duration.ofMillis(1_500));
+
+        DistributionSummary histogram = DistributionSummary.builder("test.histogram")
+                .publishPercentileHistogram()
+                .register(registry);
+        histogram.record(0.15);
+        histogram.record(15);
+        histogram.record(5E18);
+
+        DistributionSummary slos = DistributionSummary.builder("test.slos")
+                .serviceLevelObjectives(100, 200, 300)
+                .register(registry);
+        slos.record(10);
+        slos.record(250);
+        slos.record(1_000);
+
+        String scraped = registry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100);
+        assertThat(scraped).contains("my_counter_total 1.0 # {span_id=\"1\",trace_id=\"2\"} 1.0");
+        assertThat(scraped)
+                .contains("test_timer_seconds_bucket{le=\"0.1\"} 1.0 # {span_id=\"3\",trace_id=\"4\"} 1.5E7")
```

Comment:
It looks like the exemplar value is being recorded in nanoseconds. I think for Prometheus, it should be seconds to match the bucket unit. [The spec](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#histogram) doesn't explicitly talk about unit of the exemplar value, but it has:

> Each bucket covers the values less and or equal to it, and the value of the exemplar MUST be within this range. Exemplars SHOULD be put into the bucket with the highest value. A bucket MUST NOT have more than one exemplar.

Reply:
Fixed, under-the-hood the recorded values (and the buckets) are in nanos so that's what we get if we save the raw value, I converted them to seconds.